build-java/examples/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
```

Comment:
We'll discuss IRL, because my initial comment was not clear.
I suggested what you said:
> Then we can merge(copy) these files into the generated client code, and let another CI pipeline to build the generated code.
With small differences.